editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -2039,10 +2051,22 @@ VisualShaderEditor::VisualShaderEditor() {
 	add_options.push_back(AddOption("ColorConstant", "Color", "Variables", "VisualShaderNodeColorConstant", TTR("Color constant."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 	add_options.push_back(AddOption("ColorUniform", "Color", "Variables", "VisualShaderNodeColorUniform", TTR("Color uniform."), -1, VisualShaderNode::PORT_TYPE_COLOR));
 
-	// BOOLEAN
+	// CONDITIONAL
 
+	add_options.push_back(AddOption("Equal", "Conditional", "Functions", "VisualShaderNodeCompare", TTR("Returns a boolean which contained the result of (==) comparsion between two parameters."), VisualShaderNodeCompare::FUNC_EQUAL, VisualShaderNode::PORT_TYPE_BOOLEAN));
```

Comment:
Should be "which contain**s**" and "compar**is**on" in all sentences.

Reply:
Style wise, I'd suggest to spell out the comparison name directly (the "usual" names for those operators are listed in `@GlobalScope.xml`, I reviewed them a few days ago). Maybe this:
```
Returns the boolean result of the \"equality\" (==) comparison between two parameters.
Returns the boolean result of the \"greater than\" (>) comparison between two parameters.
Returns the boolean result of the \"greater than or equal\" (>=) comparison between two parameters.
```
etc.

Might be worth making it `TTR("Returns the boolean result of the %s comparison between two parameters.")` and use `vformat` to pass `TTR("\"greater than or equal\" (>=)")`, etc.

Reply:
> use vformat to pass TTR("\"greater than or equal\" (>=)"), etc.

Or maybe this to avoid confusion around the escaped quotes:
`TTR("Greater Than or Equal (>=)")`

Reply:
Done.. Check again